static/css/button.css

```diff
@@ -0,0 +1,24 @@
+#backTopBtn {
+    display: none; /* Hidden by default */
+    position: fixed; /* Fixed/sticky position */
+    bottom: 20px; /* Place the button at the bottom of the page */
+    right: 15px; /* Place the button 30px from the right */
+    z-index: 99; /* Make sure it does not overlap */
+    overflow: hidden;
+    border: none; /* Remove borders */
+    outline: none; /* Remove outline */
+    background-color: #4d647b; /* Set a background color */
```

Comment:
If you add `btn` class to `#backTopBtn` and skip the parameter `background-color`, it would be more consistent. I think.